website/docs/r/service_slack.html.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: "gitlab"
+page_title: "GitLab: gitlab_service_slack"
+sidebar_current: "docs-gitlab-resource-service_slack"
+description: |-
+  Manage Slack notifications integration that allows to receive event notifications in Slack
+---
+
+# gitlab\_project
+
+This resource allows you to manage Slack notifications integration.
+
+
+## Example Usage
+
+```hcl
+resource "gitlab_project" "awesome_project" {
+  name = "awesome_project"
+  description = "My awesome project."
+  visibility_level = "public"
+}
+
+resource "gitlab_service_slack" "slack" {
+  project                    = "${gitlab_project.awesome_project.id}"
+  webhook                    = "https://webhook.com"
+  username                   = "myuser"
+  push_events                = true
+  push_channel               = "push_chan"
+  issues_events              = true
```

Comment:
what happens if you set this and not issue_channel?

Reply:
This is valid and would work but you're right that it's not really clear, so I made the change.